scalableminds/webknossos
# Allow hotlink to organization-specific sign-up form

### URL of deployed dev instance (used for testing):
- https://signuphotlink.webknossos.xyz

### Steps to test:
- use the registration form as normal
----------------
- visit users list and click on "invite users" button to copy the shown url
- use the url to create a new account (the organization should not be visible in the form as its pre-defined via the url)
- ensure the account was created for the correct organization
----------------
- visit /onboarding to create a new organization (use a name with spaces and some special characters, such as "#")
- in the last step, click the "invite users" button, copy the link and follow the steps from above for sign-up

### Issues:
- fixes #1278

### Screenshots
![image](https://user-images.githubusercontent.com/2486553/42758968-a10bd596-8905-11e8-9908-3b274644162d.png)
---------------
![image](https://user-images.githubusercontent.com/2486553/42758973-a3fbfd6c-8905-11e8-80c6-33ad83543dcd.png)
----------------
![image](https://user-images.githubusercontent.com/2486553/42758821-10e8cc4e-8905-11e8-8186-02c41f9393dd.png)

------
